client/web/src/site-admin/SiteAdminRepositoriesPage.tsx

```diff
@@ -288,6 +288,17 @@ export const SiteAdminRepositoriesPage: React.FunctionComponent<React.PropsWithC
                 filter: { name: 'status', value: 'failed-fetch' },
             },
         ]
+        if (data.repositoryStats.corrupted > 0) {
+            items.push({
+                value: data.repositoryStats.corrupted,
+                description: 'Corrupted',
+                color: 'var(--danger)',
+                position: 'right',
+                tooltip: 'The number of repositories where corruption has been detected.',
```

Comment:
```suggestion
                tooltip: 'The number of repositories where corruption has been detected. Reclone these repositories to get rid of the corruption.',
```

We should give the admins something actionable here.